terraform/module-ses/outputs.tf

```diff
@@ -1,27 +1,28 @@
 # SES iam
 
 output "iam_ses_key" {
-  value = "${aws_iam_access_key.ses_smtp_user.id}"
+  value = aws_iam_access_key.ses_smtp_user.id
```

Comment:
since we have count on `ses_smtp_user`

You might need to change for    `value = aws_iam_access_key.ses_smtp_user[0].id`

same for the others

Reply:
I do not understand why we have a count on it ? I mean we have only one user: `ses_smtp_user` ?

Reply:
We have a count to make some resource optionnal.
For example if the user already have a dedicated smtp user, he might don't want to create one more useless user.

To be able to make it optionnal in terraform the only solution is having a count 0 or 1, but it also create an array on the resource if it is created